plugin/evm/message/handshake/upgrade_config_message_test.go

```diff
@@ -0,0 +1,44 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package handshake
+
+import (
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/precompile/contracts/nativeminter"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSerialize(t *testing.T) {
+	var t0 uint64 = 0
+	var t1 uint64 = 1
+	message, err := NewUpgradeConfigMessage(&params.UpgradeConfig{
+		PrecompileUpgrades: []params.PrecompileUpgrade{
+			{
+				Config: nativeminter.NewConfig(&t0, nil, nil, nil, nil), // enable at genesis
+			},
+			{
+				Config: nativeminter.NewDisableConfig(&t1), // disable at timestamp 1
+			},
+		},
+	})
+	require.NoError(t, err)
+
+	config, err := NewUpgradeConfigMessageFromBytes(message.Bytes())
+	require.NoError(t, err)
+
+	message2, err := NewUpgradeConfigMessage(config)
+	require.NoError(t, err)
+
+	config3, err := NewUpgradeConfigMessageFromBytes(message2.Bytes())
+	require.NoError(t, err)
+
+	message3, err := NewUpgradeConfigMessage(config3)
+	require.NoError(t, err)
```

Comment:
These seem repetitive and unnecessary to me. If message2 and message are equal, these are guaranteed to be equal, right?.

Reply:
1. `message` is from local configs
2. `message2` is from a deserialization (simulating a network exchange)
3. `message3` is to check if deserializing from a network exchange and serializing it again is consistent (generates the same hash).